src/pykiso/lib/connectors/cc_serial.py

```diff
@@ -198,6 +198,9 @@ def _cc_send(self, msg: ByteString, timeout: float = None, **kwargs) -> None:
             self.current_write_timeout = timeout
             self.serial.write_timeout = timeout
 
+        if isinstance(msg, str):
```

Comment:
Maybe add a comment on top to explain this line? The coverage says you are not verifying it.